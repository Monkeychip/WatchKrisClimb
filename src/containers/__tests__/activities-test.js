import React from 'react';
import ReactDOM from 'react-dom';
import Activities from '../activities';

test('activities test that it mounts', () => {
  const div = document.createElement('div');

});

