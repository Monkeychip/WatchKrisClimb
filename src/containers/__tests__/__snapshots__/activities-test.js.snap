// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Activities can process and return input 1`] = `
<div
  id="activities_header"
>
  Total Climbed this Year:
  <span
    id="elevation_total"
  >
    <br />
    3,280 ft
  </span>
</div>
`;
